internal/pkg/cli/flag.go

```diff
@@ -200,8 +201,9 @@ const (
 	imageTagFlagDescription     = `Optional. The container image tag.`
 	resourceTagsFlagDescription = `Optional. Labels with a key and value separated by commas.
 Allows you to categorize resources.`
-	stackOutputDirFlagDescription = "Optional. Writes the stack template and template configuration to a directory."
-	prodEnvFlagDescription        = "If the environment contains production services."
+	stackOutputDirFlagDescription  = "Optional. Writes the stack template and template configuration to a directory."
+	uploadResourcesFlagDescription = "Optional. Whether to upload dependency resources (e.g., image that needs to build and push), so that the output of `copilot svc package` can be directly used for deployment."
```

Comment:
```suggestion
	uploadResourcesFlagDescription = "Optional. Whether to upload dependency resources (e.g., an image that needs to be built and pushed), so that the output of `copilot svc package` can be directly used for deployment."
```